spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/quartz/QuartzEndpointWebExtension.java

```diff
@@ -79,6 +80,18 @@ public WebEndpointResponse<Object> quartzJobOrTrigger(SecurityContext securityCo
 				() -> this.delegate.quartzTrigger(group, name, showUnsanitized));
 	}
 
+	@WriteOperation
+	public WebEndpointResponse<Object> triggerQuartzJob(@Selector String jobs, @Selector String group,
```

Comment:
We're not fan of this approach and we don't want the action to be part of the URL. We've brainstormed quite a bit and we believe something else is missing that I am going to tackle in #43226. 

Rather than this, we'd like an approach where a `POST` on the job detail could trigger a new execution if the payload contains `"running" : true`. Can you please review the PR in that direction? It doesn't need the related issue but we'll process them both once they're ready to be merged.

Reply:
Thanks, @snicoll.

Initially, I was thinking of including `action=TRIGGER|RUN|EXECUTE` as part of the request body.

I have one more question regarding `running:true`. What should be returned if someone provides `running:false`? Should it result in a bad request or a not-found response?

Reply:
I was thinking about `action` because quartz supports `pause`, `resume` actions a well.

Reply:
This feature might be requested in the future. For instance, if a job is broken or encountering issues, someone might want the ability to pause it.

Reply:
The idea is that you `POST` a body indicating the desired state. For a job, posting `running: true` indicates that the job should be run now. AFAIK, Quartz doesn't support cancellation of a running job so posting `running:false` would be a bad request.

AFAIK, pausing is supported by triggers not jobs so I'd expect that support to be added to a resource that represents a trigger. You'd post something like `"state": "paused"` to pause a trigger and `"state": "normal"` to resume it.

Reply:
Apologies, I misunderstood the `running: false/true part`. You intend to include this information in a detailed response to inform users that the job can be triggered using the **POST** method `/actuator/quartz/jobs/{groupName}/{jobName}`.
Am I right?

Reply:
> AFAIK, pausing is supported by triggers not jobs so I'd expect that support to be added to a resource that represents a trigger. You'd post something like "state": "paused" to pause a trigger and "state": "normal" to resume it.
 
 
 ```java
 
     /**
     * Pause the <code>{@link org.quartz.JobDetail}</code> with the given
     * key - by pausing all of its current <code>Trigger</code>s.
     * 
     * @see #resumeJob(JobKey)
     */
    void pauseJob(JobKey jobKey)
        throws SchedulerException;
        
            /**
     * Resume (un-pause) the <code>{@link org.quartz.JobDetail}</code> with
     * the given key.
     * 
     * <p>
     * If any of the <code>Job</code>'s<code>Trigger</code> s missed one
     * or more fire-times, then the <code>Trigger</code>'s misfire
     * instruction will be applied.
     * </p>
     * 
     * @see #pauseJob(JobKey)
     */
    void resumeJob(JobKey jobKey)
        throws SchedulerException;
        
 ```

Reply:
To be honest, I do like `"state": "running" ` more than  `"running": "true" `